plasmapy/utils/decorators/tests/test_helpers.py

```diff
@@ -1,14 +1,114 @@
 """Tests for module :mod:`plasmapy.utils.decorators`."""
 import inspect
+import pytest
 
 from unittest import mock
 
-from ..helpers import preserve_signature
+from ..helpers import modify_docstring, preserve_signature
 
 
-# ----------------------------------------------------------------------------------------
+# --------------------------------------------------------------------------------------
+# Test Decorator `modify_docstring`
+# --------------------------------------------------------------------------------------
+class TestModifyDocstring:
+    # create function to test on
+    @staticmethod
+    def foo_simple(x: float, y: float) -> float:
+        """A simple docstring."""
+        return x + y
+
+    @staticmethod
+    def foo_complex(x: float, y: float) -> float:
```

Comment:
```suggestion
    def simple_docstring(x: float, y: float) -> float:
        """A simple docstring."""
        return x + y

    @staticmethod
    def complex_docstring(x: float, y: float) -> float:
```
I was thinking that the intent would be more clear with something like these names.